packages/cli/deno.json

```diff
@@ -4,7 +4,8 @@
     "cli": "ROOT=$(pwd) && cd $INIT_CWD && deno run --allow-net --allow-ffi --allow-read --allow-write --allow-env \"$ROOT/mod.ts\"",
     "cli-no-pwd-override": "deno run --allow-net --allow-ffi --allow-read --allow-write --allow-env ./mod.ts",
     "test": "deno test --allow-ffi --allow-read --allow-write --allow-run --allow-env test/*.test.ts",
-    "integration": "CT_CLI_INTEGRATION_USE_LOCAL=1 LOG_LEVEL=warn ./integration/integration.sh"
+    "integration": "CT_CLI_INTEGRATION_USE_LOCAL=1 LOG_LEVEL=warn ./integration/integration.sh",
+    "acl-integration": "CT_CLI_INTEGRATION_USE_LOCAL=1 LOG_LEVEL=warn ./integration/acl.sh"
```

Comment:
This isn't run anywhere yet, as the assertions here aren't yet true, but cover the e2e scenario